third_party/rust/chromium_crates_io/Cargo.toml

```diff
@@ -218,3 +218,15 @@ package = "winapi"
 [patch.crates-io.zcash_v1]
 path = "../../../components/brave_wallet/browser/zcash/rust"
 package = "zcash"
+
+[patch.crates-io.zcash_protocol_v0_1]
```

Comment:
you should not be adding these here, they are not be used as crates, they are just rust sources in a rust static lib